src/streamsaver.ts

```diff
@@ -0,0 +1,11 @@
+import streamSaver from 'streamsaver';
+import { WritableStreamIsNative, WritableStreamPonyfill } from './streams';
+
+streamSaver.mitm = 'https://streaming.transcend.io/mitm.html';
+
+if (!WritableStreamIsNative) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (streamSaver as any).WritableStream = WritableStreamPonyfill;
```

Comment:
curious: Is this supposed to say Ponyfill?

Reply:
Yep